draft-ietf-mls-architecture.md

```diff
@@ -463,7 +460,8 @@ goals ("MLS aims to guarantee that...")?]]
 We assume that all transport connections are secured via some transport
 layer security mechanism such as TLS {{?I-D.ietf-tls-tls13}}. However,
 as noted above, the security of MLS will generally survive compromise
-of the transport layer.
+of the transport layer. Note that this may not extend to connections with the
+AS, which will typically require authentication at a minimum.
```

Comment:
What do you mean here ?

Reply:
We claim the MLS should be secure even without any sort of transport security. This makes sense for the DS, but if we are to have an AS which authoritatively authenticates identities, this will have to be authenticated.